src/sentry/api/endpoints/project_rules.py

```diff
@@ -49,7 +53,28 @@ def post(self, request, project):
         serializer = RuleSerializer(context={"project": project}, data=request.data)
 
         if serializer.is_valid():
-            rule = serializer.save(rule=Rule())
+            data = serializer.validated_data
+            kwargs = {
+                "name": data["name"],
+                "environment": data.get("environment"),
+                "project": project,
+                "action_match": data["actionMatch"],
+                "conditions": data["conditions"],
+                "actions": data["actions"],
+                "frequency": data.get("frequency"),
+            }
+
+            rule = project_rules.Creator.run(
+                pending_save=data.get("pending_save", False), request=request, **kwargs
+            )
+
+            if not rule:
```

Comment:
@MeredithAnya Probably worth adding a comment that the only reason that the creator would not return a rule is because we are doing the async job.

Reply:
@MeredithAnya  Actually...why do we call the Creator at all when `pending_save=true`? If all the creator does in that case is quit, I think it makes to just not call the creator.